Project/Code/Converter/TilesetConverterVerticalRM2K3.cs

```diff
@@ -90,17 +90,22 @@ private Bitmap[] SetMVAutotile(Bitmap bmp)
             for (int i = 0; i < autotiles.Length; i++)
             {
                 Bitmap temp = new Bitmap(96, 144);
-                Bitmap bmp1 = Stretch(Crop(autotiles[i], 16, 0, 16, 16), Maker.MV_A12.SPRITE_SIZE);
-                Bitmap bmp2 = Stretch(Crop(autotiles[i], 32, 0, 16, 16), Maker.MV_A12.SPRITE_SIZE);
-                Bitmap bmp3 = Stretch(Crop(autotiles[i], 0, 16, 48, 48), 96);
-                temp = Paste(temp, bmp1, 0, 0, Maker.MV_A12.SPRITE_SIZE, Maker.MV_A12.SPRITE_SIZE);
-                temp = Paste(temp, bmp2, 48, 0, Maker.MV_A12.SPRITE_SIZE, Maker.MV_A12.SPRITE_SIZE);
-                temp = Paste(temp, bmp3, 0, 48, 96, 96);
+                Bitmap singleTileBmp = Stretch(Crop(autotiles[i], 0, 0, 16, 16), Maker.MV_A12.TILE_SIZE);
```

Comment:
This line was the cause of the wrong visualisation.
Changed 16 to 0.
Also adjusted unit test and gave better naming to some variabled related to these parts.